Status Reports/Readme.md

```diff
@@ -7,14 +7,14 @@ Status Report Template: https://github.com/openmainframeproject-internship/resou
 
 | Document | Description | Document Name |
 |---|---|---|
-| Project Plan | Overall week by week plan of milestones | |
-| Week 1 Status Report | | | 
-| Week 2 Status Report | | |
-| Week 3 Status Report | | |
-| Week 4 Status Report | | |
-| Week 5 Status Report | | |
-| Week 6 Status Report | | |
-| Week 7 Status Report | | |
+| Project Plan | Overall week by week plan of milestones | [project plan.md](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/project%20plan.md) |
+| Week 1 Status Report | Work done in week 1 | [Week 1 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%201%20Status%20Report.docx) | 
+| Week 2 Status Report | Work done in week 2 |[Week 2 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%202%20Status%20Report.docx) |
+| Week 3 Status Report | Work done in week 3 |[Week 3 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%203%20Status%20Report.docx) |
+| Week 4 Status Report | Work done in week 4 |[Week 4 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%204%20Status%20Report.docx) |
+| Week 5 Status Report | Work done in week 5 |[Week 5 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%205%20Status%20Report.docx) |
+| Week 6 Status Report | Work done in week 6 |[Week 6 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%206%20Status%20Report.docx) |
+| Week 7 Status Report | Work done in week 7 |[Week 7 Status Report.docx](https://github.com/muhammad-md/Zowe-explorer-extension-templates/blob/master/Status%20Reports/Week%207%20Status%20Report.docx) |
```

Comment:
thanks for adding these.  

i recognize that the template suggested windows word document files, in practice though, on my development teams, we typically use markdown since it's rendered right in the browser.  otherwise, we'd have to download the docx files and open them locally.